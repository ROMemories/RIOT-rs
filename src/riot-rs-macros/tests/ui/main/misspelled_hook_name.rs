#![no_main]
#![feature(type_alias_impl_trait)]
#![feature(used_with_arg)]

// As the macro will fail, this import will not get used
#[allow(unused_imports)]
use riot_rs::embassy::usb::UsbBuilderHook;

// FAIL: misspelled hook name
#[riot_rs::main(usb_builder_hooook)]
async fn main() {}
