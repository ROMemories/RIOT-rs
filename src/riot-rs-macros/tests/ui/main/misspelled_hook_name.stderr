error: unsupported hook (`usb_builder` are supported)
  --> tests/ui/main/misspelled_hook_name.rs:13:23
   |
13 | #[riot_rs::main(hooks(usb_buer))]
   |                       ^^^^^^^^
